lib/service/service_test.go

```diff
@@ -235,20 +230,17 @@ func (s *ServiceTestSuite) TestCheckPrincipals(c *check.C) {
 			outRegenerate: false,
 		},
 	}
-	for _, tt := range tests {
-		ok := checkServerIdentity(testConnector, tt.inPrincipals, tt.inDNS, t.Log)
-		c.Assert(ok, check.Equals, tt.outRegenerate)
+	for i, tt := range tests {
+		ok := checkServerIdentity(testConnector, tt.inPrincipals, tt.inDNS, logrus.New().WithField("test", "TestServiceCheckPrincipals"))
+		require.Equal(t, tt.outRegenerate, ok, "test %d", i)
 	}
 }
 
-// TestInitExternalLog verifies that external logging can be used both as a means of
+// TestServiceInitExternalLog verifies that external logging can be used both as a means of
 // overriding the local audit event target.  Ideally, this test would also verify
 // setup of true external loggers, but at the time of writing there isn't good
 // support for setting up fake external logging endpoints.
-func (s *ServiceTestSuite) TestInitExternalLog(c *check.C) {
-	t := testlog.NewCheckTestWrapper(c)
-	defer t.Close()
-
+func TestServiceInitExternalLog(t *testing.T) {
```

Comment:
I realize that you were simply maintaining existing functionality here, but I feel like this test could be simplified by converting `isNil` and `isErr` to `require.ValueAssertionFunc` and `require.ErrorAssertionFunc` and explicitly setting them during initialization.